lib/asciidoctor/pdf/theme_loader.rb

```diff
@@ -163,7 +163,12 @@ def process_entry key, val, data, normalize_key = false
           end
         elsif key.end_with? '_color'
           # QUESTION: do we really need to evaluate_math in this case?
-          data[key] = to_color evaluate val, data
+          val = evaluate val, data
+          if key == 'table_grid_color' && ::Array === val && val.size == 2 # x,y
+            data[key] = [to_color(val[0]), to_color(val[1])]
```

Comment:
The style in Asciidoctor is to put the round brackets around the method call. So this would be:

```rb
data[key] = [(to_color val[0]), (to_color val[1])]
```

RuboCop doesn't always catch it.

But to make it more clear, I prefer:

```
data[key] = val.map {|e| to_color e }
```